src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -191,13 +198,18 @@
                                                                                 // Prepare a new network interface (one per VM)
                                                                                 String networkInterfaceName = createUniqueNetworkInterfaceName(createUniqueInstanceTag(instanceTag,
                                                                                                                                                                        instanceNumber));
-                                                                                Creatable<NetworkInterface> creatableNetworkInterface = azureProviderUtils.prepareNetworkInterfaceFromScratch(azureService,
-                                                                                                                                                                                              region,
-                                                                                                                                                                                              resourceGroup,
-                                                                                                                                                                                              networkInterfaceName,
-                                                                                                                                                                                              creatableVirtualNetwork,
-                                                                                                                                                                                              creatableNetworkSecurityGroup,
-                                                                                                                                                                                              creatablePublicIPAddress);
+
```

Comment:
Can you create a private method and use it in the lambda ? This is becoming too long it is unreadable.